cro-clib/chain.h

```diff
@@ -69,27 +108,28 @@ CroResult cro_destroy_hdwallet(CroHDWalletPtr hdwallet);
 
 /**
  * print address information
```

Comment:
can you update the comments? they all say "print address information"

Reply:
ok